packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,113 @@
+  component: Loading,
 
-Default.story = {
   parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
   },
-};
-
-export const ButtonLoading = () => <Button loading>Default button</Button>;
 
-ButtonLoading.story = {
-  name: "Button loading",
-
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  args: {
+    loading: true,
+    text: <Text>Please wait, content is loading...</Text>,
+    customSize: 50,
+    type: TYPE_OPTIONS.PAGE_LOADER,
+    id: "loader-id",
   },
-};
 
-export const CardLoading = ({ title, description, loading, loadingText }) => {
-  return (
-    <Card loading={loading} title={title} description={description}>
-      <Loading loading={loading} type="boxLoader" text={loadingText}>
-        <CardSection>
-          <Illustration name="EnjoyApp" size="medium" />
-        </CardSection>
-      </Loading>
-    </Card>
-  );
-};
-
-CardLoading.story = {
-  name: "Card loading",
-
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
   },
 };
 
-CardLoading.args = {
-  title: "Card with title",
-  description: "",
-  loading: true,
-  loadingText: "Please wait, Card content is loading...",
-};
+export default meta;
 
-export const InlineLoader = ({ loadingText }) => {
-  return <Loading type={TYPE_OPTIONS.INLINE_LOADER} text={loadingText} />;
-};
+type Story = StoryObj<typeof Loading>;
 
-InlineLoader.story = {
-  name: "Inline loader",
+export const Default: Story = {
+  args: {
+    customSize: undefined,
+    loading: undefined,
+    text: undefined,
+    type: undefined,
+  },
 
   parameters: {
-    info: "This configuration of this component is without any height, width or paddings. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-InlineLoader.args = {
-  loadingText: "Please wait, content of the page is loading...",
-};
+export const DefaultWithChildren: Story = {
+  render: args => (
+    <Loading {...args}>
+      <Text>Loading prop is false.</Text>
+    </Loading>
+  ),
 
-export const CustomLoader = ({ customSize }) => {
-  return <Loading customSize={customSize} />;
-};
+  parameters: {
+    controls: {
+      exclude: ["children", "customSize", "text", "type"],
+    },
+  },
 
-CustomLoader.args = {
-  customSize: 50,
+  args: {
+    customSize: undefined,
+    text: undefined,
+    type: undefined,
+  },
 };
 
-export const Playground = ({ type, loadingText, loading, dataTest }) => {
-  return <Loading loading={loading} type={type} text={loadingText} dataTest={dataTest} />;
-};
+export const Playground: Story = {
```

Comment:
Card component story - I removed it as I found it redundant because the Card wrapper just wraps `loading` component and the children is the same as `Default with children` story.

`loading` missing in playground - the example in Playground story doesn't have any children and the behaviour is same for `false`/`true` value for `loading` prop.

text in Playground - this prop is `React.ReactNode` type, the changes text string need to be in `"`, either it's not working. I was thinking it can be confusing for the user and having the `text` as a constant ReactNode element in the example code is more clear (?). Idk, what would be better, what's your opinion?

![image](https://github.com/user-attachments/assets/6835dd30-8860-468b-ab13-30f5a03fccd5)

![image](https://github.com/user-attachments/assets/2d613031-0821-4e1c-aa3f-64edb8bdb3e2)

Reply:
I am also wondering if it's not better to let there static `<Text>` element as it follows the prop type of <Loading> component. 🤔

Reply:
ok, we can keep card as an example. 👍